mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -34,6 +34,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{StructField, StructType}
+import org.apache.spark.util.VersionUtils._
```

Comment:
got your point. I agree. I will change the importing to 
import org.apache.spark.util.VersionUtils.majorVersion instead for clarity.